src/org/thoughtcrime/securesms/ProfileSettingsFragment.java

```diff
@@ -248,8 +258,7 @@ private void onNewChat() {
     @Override
     public boolean onCreateActionMode(ActionMode mode, Menu menu) {
       mode.getMenuInflater().inflate(R.menu.profile_context, menu);
-      DcChat dcChat = dcContext.getChat(chatId);
-      menu.findItem(R.id.delete).setVisible(!dcChat.isMailingList());
```

Comment:
this is not necessary since the member list is not even shown in a mailing list's profile so if we are inside `onCreateActionMode` this can not be a mailing list